lib/assert.mjs

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-restricted-properties */
```

Comment:
Just a thought... could we not devise a mechanism for generating these automatically during the build so we do not need to keep them manually in sync? The surface area of core modules is fixed at time of build.

Reply:
@jasnell we could somehow with static parsing maybe, but various things like getter/setters would need to be excluded.